### PR TITLE
update verify to 13.3.1

### PR DESCRIPTION
**Pull request by SimonCropp**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6090)

### CODE REVIEW

**Comment by RussKie** [comment]
Thank you. We don't use nuget.org, and use pre-vetted packages from internal feeds. Thus we don't have access to the latest versions, and need to request those.
image

**Comment by SimonCropp** [comment]
i was experimenting is the line

```
<!-- This is needed for Verify.Xunit to pull correct version of System.Speech -->
<MicrosoftWindowsCompatibilityVersion>7.0.0-alpha.1.21528.1</MicrosoftWindowsCompatibilityVersion>
```

this should no longer be needed by Verify. But is seems other stuff unrelated to Verify requires it? Perhaps it should be moved somewhere else?

**Comment by SimonCropp** [comment]
> Thank you. We don't use nuget.org, and use pre-vetted packages from internal feeds. Thus we don't have access to the latest versions, and need to request those.

Anything i can help with there?

**Comment by RussKie** [comment]
No, it is on us. We're currently under the pump with releases, but we'll action it in the near future.

**Comment by RussKie** [comment]
> i was experimenting is the line
> 
> ```
> 
> <MicrosoftWindowsCompatibilityVersion>7.0.0-alpha.1.21528.1</MicrosoftWindowsCompatibilityVersion>
> ```
> 
> this should no longer be needed by Verify. But is seems other stuff unrelated to Verify requires it? Perhaps it should be moved somewhere else?

It'd be great if it's no longer needed. There was some transitive dependency issue that required this.

**Comment by SimonCropp** [comment]
ok cool. want me to leave this open to track? or close it?

**Comment by RussKie** [comment]
Please keep it open, this way it would be hard for us to forget about it 😊

**Comment by RussKie** [comment]
Raised a request to @dotnet/dnceng to load the latest version (14.2) to the dotnet-public feed.

**Comment by RussKie** [comment]
I need the latest version to enable certain test cases for https://github.com/dotnet/winforms/issues/6144

**Comment by epananth** [comment]
Updated to the latest version https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-public%40Local&package=Verify.Xunit&version=14.2.0&protocolType=NuGet

**Comment by RussKie** [comment]
@epananth @dotnet/dnceng there are few dependencies that need to be updated as well:

*	https://www.nuget.org/packages/Verify/14.2
*	https://www.nuget.org/packages/EmptyFiles/2.8
*	https://www.nuget.org/packages/DiffEngine/8.3
*	https://www.nuget.org/packages/SimpleInfoName/1.1.1

File: eng/Versions.props [hunk 1]
```diff
@@ -66,9 +66,7 @@
   </PropertyGroup>
   <!-- Test related -->
   <PropertyGroup>
-    <VerifyXunitVersion>11.18.2</VerifyXunitVersion>
-    <!-- This is needed for Verify.Xunit to pull correct version of System.Speech -->
-    <MicrosoftWindowsCompatibilityVersion>7.0.0-alpha.1.21528.1</MicrosoftWindowsCompatibilityVersion>
```
Comment (RussKie) [inline eng/Versions.props:71]: I tried v14.2 locally and it still doesn't work
```
D:\Development\dotnet-winforms\src\System.Windows.Forms.Analyzers.CSharp\tests\UnitTests\System.Windows.Forms.Analyzers.CSharp.Tests.csproj : error NU1604: Project dependency Microsoft.Windows.Compatibility does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results. [D:\Development\dotnet-winforms\Winforms.sln]
```

Comment (SimonCropp) [reply eng/Versions.props:71]: yep. but (i dont think) this is caused by the verify reference.

**Comment by RussKie** [comment]
Obsolete by https://github.com/dotnet/winforms/issues/6144

**Pull request closed without merging**